### DISCUSSION TITLE
Input model validation?

### DISCUSSION THREAD

**Discussion by wondering639** [ChilliCream/hotchocolate#4717] [Q&A] [answered]
Is there something like automatic (input) model validation in HotChocolate, like e.g. WebAPI does it with REST? Otherwise, what's the best practice to validate inputs?

**Comment by PascalSenn** [comment, marked as answer]
Checkout the community packages: https://github.com/ChilliCream/hotchocolate/blob/main/COMMUNITY.md#validation

**Reply by wondering639** [reply]
thanks for that pointer :-)
